src/platform_impl/macos/mod.rs

```diff
@@ -311,6 +326,7 @@ impl MenuChild {
             id: COUNTER.next(),
             native_icon,
             accelerator,
+            children: None,
```

Comment:
is there a reason for this? it should be covered by `..Default::default`, no?

Reply:
my bad, i was testing this compiler error i never faced https://doc.rust-lang.org/error_codes/E0509.html